langs/kotlin.go

```diff
@@ -163,8 +166,15 @@ func kotlinMavenOpts() string {
 
 /*    TODO temporarily generate maven project boilerplate from hardcoded values.
       Will eventually move to using a maven archetype.*/
-func kotlinPomFileContent(APIversion string) string {
-	return fmt.Sprintf(kotlinPomFile, APIversion)
+func kotlinPomFileContent(APIversion, groupId, pomType string) string {
+	if groupId == "io.fnproject.com" {
```

Comment:
Similarly I think this case can be merged with the pomType == maven case.

Reply:
Done